cfpb/hmda-frontend
# [WIP] Display release version

## Testing

1. `yarn start`
1. Observe version displayed in homepage footer
1. Observe version displayed in Filing homepage footer
1. In second terminal, run `yarn release 2.0.0`
1. Observe updated version

## Should we include

- the option to perform Docker build and tag?
- the option to push to image repo? 
- the option to `git tag <ver> && git push origin <ver>` release?
